renovate.json

```diff
@@ -9,5 +9,5 @@
   "rebaseStalePrs": false,
   "rangeStrategy": "bump",
   "bumpVersion": null,
-  "semanticCommitScope": "starters"
+  "semanticCommitScope": null
```

Comment:
@pieh This is because PRs updating dependencies on theme packages at the moment still say `chore(starters): blah` 🤦‍♂ 

With this all PRs from Renovate will say `chore: blah`